### ISSUE TITLE
zipline example data currently has to be built in py3

### ISSUE THREAD

**Issue by jbredeche** [quantopian/zipline#1229]
If you build it in py2, you get bytes/str decoding errors in py3.  Would be nice to have it work in both envs no matter which env was used to build it.

**Comment by llllllllll** [comment]
@ssanderson seemed to be able to run this in py2, can you compare `pip freeze` to see what else may be causing this?

**Comment by ssanderson** [comment]
@llllllllll I think the bug @jbredeche hit was "build example data in py2, then try to use it in py3".  Writing and running solely in py2 seems to work fine; the issue I'm guessing is authoring in py2 and then running in py3.

**Comment by ssanderson** [comment]
(Just confirmed on my machine that this fails if I write in py2 and read in py3).

**Comment by jbredeche** [comment]
Yup, @ssanderson described my issue correctly. Apologies for the ambiguity.
